<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="RedLineScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>411</width>
      <height>481</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="RedLineLabels">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>409</width>
       <height>479</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="RedLineLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt</string>
    </property>
    <property name="text">
     <string>Red Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GreenLineLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt</string>
    </property>
    <property name="text">
     <string>Green Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="GreenLineScrollArea">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>411</width>
      <height>481</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="RedLineLabels_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>409</width>
       <height>479</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="RedLineHeadersLayout">
     <item>
      <widget class="QLabel" name="RedSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Section</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RedTrainCount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string># of Trains</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RedOcc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Blocks Occcupied</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RedInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="Spacer1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>16</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="Spacer2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>16</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="RedLineHeadersLayout_2">
     <item>
      <widget class="QLabel" name="GreenSection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Section</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="GreenTrainCount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string># of Trains</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="GreenOcc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Blocks Occcupied</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="GreenInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="GeneralInfoLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt</string>
    </property>
    <property name="text">
     <string>General Info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DateTimeLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date/Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenLineGenInfo">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>90</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt;
background-color: #00b321;
font-weight: bold</string>
    </property>
    <property name="text">
     <string>Green Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RedLineGenInfo">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt;
background-color: #FF0000;
font-weight: bold</string>
    </property>
    <property name="text">
     <string>Red Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RailBreakagesLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
background-color: #a8a8a8;
color: black</string>
    </property>
    <property name="text">
     <string>Rail Breakages</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="GeneralInfoSpacer">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>180</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="RedBreakagesScroll">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>129</width>
       <height>89</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="GreenBreakagesScroll">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>180</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>129</width>
       <height>89</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TotalTrainCountLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>280</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
background-color: #a8a8a8;
color: black</string>
    </property>
    <property name="text">
     <string>Total Trains Active</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RedTrainCt">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GreenTrainCt">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>320</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RRXingLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>360</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
background-color: #a8a8a8;
color: black</string>
    </property>
    <property name="text">
     <string>RR Crossings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="I47">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>400</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>I-47:</string>
    </property>
   </widget>
   <widget class="QLabel" name="E19">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>400</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-19</string>
    </property>
   </widget>
   <widget class="QLabel" name="I47Status">
    <property name="geometry">
     <rect>
      <x>917</x>
      <y>400</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(188, 6, 0);
color: rgb(148, 0, 17);
background-color: rgb(255, 135, 119)</string>
    </property>
    <property name="text">
     <string>Inactive</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="E19Status">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>400</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(188, 6, 0);
color: rgb(148, 0, 17);
background-color: rgb(255, 135, 119)</string>
    </property>
    <property name="text">
     <string>Inactive</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadTrack">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>590</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Track Layout</string>
    </property>
   </widget>
   <widget class="QLabel" name="TrackHeatersLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
background-color: #a8a8a8;
color: black</string>
    </property>
    <property name="text">
     <string>Track Heaters</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FaultsLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>510</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16pt;
background-color: #a8a8a8;
color: black</string>
    </property>
    <property name="text">
     <string>Faults</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentTempLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Temp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeaterStatus">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>480</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(188, 6, 0);
color: rgb(148, 0, 17);
background-color: rgb(255, 135, 119)</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="FaultSpacer1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>540</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="FaultSpacer2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>540</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="PowerFaultLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>550</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BrokenRailLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Broken Rail</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BrokenCircuitLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>550</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Broken Circuit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="FaultWindowButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>550</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>957</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tempEntry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tempLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Temperature:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tempGo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>620</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="Line" name="Spacer3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="brokenRailButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>620</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Broken Rail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerFailure">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>620</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Failure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circuitFailure">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Circuit Failure</string>
    </property>
   </widget>
   <widget class="QLabel" name="brokenRailLineLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="brokenRailBlockLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Block</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brokenRailLineSelect">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="brokenRailBlockSelect">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <zorder>GeneralInfoSpacer</zorder>
   <zorder>RedLineScrollArea</zorder>
   <zorder>RedLineLabel</zorder>
   <zorder>GreenLineLabel</zorder>
   <zorder>GreenLineScrollArea</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>Spacer1</zorder>
   <zorder>Spacer2</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>GeneralInfoLabel</zorder>
   <zorder>DateTimeLabel</zorder>
   <zorder>GreenLineGenInfo</zorder>
   <zorder>RedLineGenInfo</zorder>
   <zorder>RailBreakagesLabel</zorder>
   <zorder>RedBreakagesScroll</zorder>
   <zorder>GreenBreakagesScroll</zorder>
   <zorder>TotalTrainCountLabel</zorder>
   <zorder>RedTrainCt</zorder>
   <zorder>GreenTrainCt</zorder>
   <zorder>RRXingLabel</zorder>
   <zorder>I47</zorder>
   <zorder>E19</zorder>
   <zorder>I47Status</zorder>
   <zorder>E19Status</zorder>
   <zorder>UploadTrack</zorder>
   <zorder>TrackHeatersLabel</zorder>
   <zorder>FaultsLabel</zorder>
   <zorder>CurrentTempLabel</zorder>
   <zorder>HeaterStatus</zorder>
   <zorder>FaultSpacer1</zorder>
   <zorder>FaultSpacer2</zorder>
   <zorder>PowerFaultLabel</zorder>
   <zorder>BrokenRailLabel</zorder>
   <zorder>BrokenCircuitLabel</zorder>
   <zorder>FaultWindowButton</zorder>
   <zorder>time</zorder>
   <zorder>tempEntry</zorder>
   <zorder>tempLabel</zorder>
   <zorder>tempGo</zorder>
   <zorder>Spacer3</zorder>
   <zorder>brokenRailButton</zorder>
   <zorder>powerFailure</zorder>
   <zorder>circuitFailure</zorder>
   <zorder>brokenRailLineLabel</zorder>
   <zorder>brokenRailBlockLabel</zorder>
   <zorder>brokenRailLineSelect</zorder>
   <zorder>brokenRailBlockSelect</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
